_posts/2022-05-12-data-pipelines-in-Databricks.markdown

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: "Data pipelines in Databricks"
+date: 2022-05-12
+tags: Databricks data testing
+author: Natalie Laing
+image: /img/code.jpg
+---
+
+# Data pipelines in Databricks
+
+## Introduction
+
+![Databricks logo](/img/blog/databricks-data-pipeline/databricks-logo.png)
+
+I was recently working for a client that required a data pipeline in Databricks that would conform to the DevOps way of working. This pipeline would need to have an RBAC model to conform to least privilege, monitoring and observability and include testing of transformed data.
```

Comment:
I was recently working for a client that required creating data pipelines in Databricks using software engineering good practices including infrastructure as code, testing and observability. How did we achieve this? In this blog post I will bring you along on our journey of how we were able to implement this data pipelines.